src/meson.build

```diff
@@ -2,17 +2,17 @@ nvapi_src = files([
   'util/util_string.cpp',
   'util/util_env.cpp',
   'util/util_log.cpp',
-  'sysinfo/vulkan.cpp',
-  'sysinfo/nvml.cpp',
-  'sysinfo/nvapi_output.cpp',
-  'sysinfo/nvapi_adapter.cpp',
-  'sysinfo/nvapi_adapter_registry.cpp',
+  'nvapi/vulkan.cpp',
+  'nvapi/nvml.cpp',
+  'nvapi/nvapi_output.cpp',
+  'nvapi/nvapi_adapter.cpp',
+  'nvapi/nvapi_adapter_registry.cpp',
   'resource_factory.cpp',
-  'd3d/lfx.cpp',
-  'd3d/nvapi_d3d_instance.cpp',
-  'd3d/nvapi_d3d_low_latency_device.cpp',
-  'd3d11/nvapi_d3d11_device.cpp',
-  'd3d12/nvapi_d3d12_device.cpp',
+  'nvapi/lfx.cpp',
+  'nvapi/nvapi_d3d_instance.cpp',
+  'nvapi/nvapi_d3d_low_latency_device.cpp',
+  'nvapi/nvapi_d3d11_device.cpp',
+  'nvapi/nvapi_d3d12_device.cpp',
```

Comment:
Perhaps the stuff below could also be moved to `nvapi/` (and `nvofapi{,_d3d12,_vulkan},cpp` to `nvofapi/`)? Then we could have each library be built from sources only in its own directory plus some shared utils/interfaces and this `meson.build` would just define variables for shared sources and then do `subdir('nvapi')` and `subdir('nvofapi')`.

Reply:
Sounds good. Ok to wait with this until we have an idea how the tests plays out?

Reply:
Yup!